src/Modal/Modal.jsx

```diff
@@ -5,17 +5,22 @@ require("./Modal.less");
 var DEFAULT_WIDTH = 400;
 
 var Modal = React.createClass({
+  propTypes: {
+    closeModal: React.PropTypes.func.isRequired,
+    title: React.PropTypes.string,
+    width: React.PropTypes.number,
+  },
   render: function() {
     var width = this.props.width || DEFAULT_WIDTH;
+    var modalStyles = {
+      width: width + "px",
+      marginLeft: "-" + (width / 2) + "px"
+    };
+
     return (
       <div className="Modal">
         <div className="Modal--background" onClick={this.props.closeModal}/>
-        <div
-          className="Modal--window"
-          style={{
-            width: width + "px",
-            marginLeft: "-" + (width / 2) + "px"
-          }}>
+        <div className="Modal--window" style={modalStyles}>
           <div className="Modal--window--container">
             <h2>{this.props.title}</h2>
```

Comment:
Could lead to inconsistencies in header size. But yeah there might be cases where we want an icon/graphic next to the title and it would be hard to add. Maybe we could change this to be 
```
{this.props.title && <h2>{this.props.title}</h2>} 
```
so that you could leave it out and supply your own if you want